 Network rinha-2024q1  Creating
 Network rinha-2024q1  Created
 Container postgres  Creating
 Container postgres  Created
 Container deno-api-1  Creating
 Container deno-api-2  Creating
 api-2 The requested image's platform (linux/arm64) does not match the detected host platform (linux/amd64/v4) and no specific platform was requested 
 api-1 The requested image's platform (linux/arm64) does not match the detected host platform (linux/amd64/v4) and no specific platform was requested 
 Container deno-api-2  Created
 Container deno-api-1  Created
 Container arrudaricardo-01-caddy-1  Creating
 Container arrudaricardo-01-caddy-1  Created
Attaching to caddy-1, deno-api-1, deno-api-2, postgres
postgres    | The files belonging to this database system will be owned by user "postgres".
postgres    | This user must also own the server process.
postgres    | 
postgres    | The database cluster will be initialized with locale "en_US.utf8".
postgres    | The default database encoding has accordingly been set to "UTF8".
postgres    | The default text search configuration will be set to "english".
postgres    | 
postgres    | Data page checksums are disabled.
postgres    | 
postgres    | fixing permissions on existing directory /var/lib/postgresql/data ... ok
postgres    | creating subdirectories ... ok
postgres    | selecting dynamic shared memory implementation ... posix
postgres    | selecting default max_connections ... 100
postgres    | selecting default shared_buffers ... 128MB
postgres    | selecting default time zone ... Etc/UTC
postgres    | creating configuration files ... ok
deno-api-1  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
postgres    | running bootstrap script ... ok
caddy-1     | {"level":"info","ts":1708056203.9171948,"msg":"using provided configuration","config_file":"/etc/caddy/Caddyfile","config_adapter":"caddyfile"}
caddy-1     | {"level":"warn","ts":1708056203.9183877,"msg":"Caddyfile input is not formatted; run 'caddy fmt --overwrite' to fix inconsistencies","adapter":"caddyfile","file":"/etc/caddy/Caddyfile","line":3}
caddy-1     | {"level":"warn","ts":1708056203.9185252,"logger":"admin","msg":"admin endpoint disabled"}
caddy-1     | {"level":"warn","ts":1708056203.9186232,"logger":"http.auto_https","msg":"automatic HTTPS is completely disabled for server","server_name":"srv0"}
caddy-1     | {"level":"info","ts":1708056203.9187193,"logger":"tls.cache.maintenance","msg":"started background certificate maintenance","cache":"0xc0004c6a00"}
caddy-1     | {"level":"info","ts":1708056203.919045,"logger":"http.log","msg":"server running","name":"srv0","protocols":["h1","h2","h3"]}
caddy-1     | {"level":"info","ts":1708056203.9193134,"msg":"autosaved config (load with --resume flag)","file":"/config/caddy/autosave.json"}
caddy-1     | {"level":"info","ts":1708056203.919325,"msg":"serving initial configuration"}
caddy-1     | {"level":"info","ts":1708056203.923686,"logger":"tls","msg":"cleaning storage unit","storage":"FileStorage:/data/caddy"}
caddy-1     | {"level":"info","ts":1708056203.9238343,"logger":"tls","msg":"finished cleaning storage units"}
deno-api-1 exited with code 0
deno-api-2 exited with code 0
postgres    | performing post-bootstrap initialization ... ok
postgres    | syncing data to disk ... ok
postgres    | 
postgres    | 
postgres    | Success. You can now start the database server using:
postgres    | 
postgres    |     pg_ctl -D /var/lib/postgresql/data -l logfile start
postgres    | 
postgres    | initdb: warning: enabling "trust" authentication for local connections
postgres    | initdb: hint: You can change this by editing pg_hba.conf or using the option -A, or --auth-local and --auth-host, the next time you run initdb.
postgres    | waiting for server to start....2024-02-16 04:03:25.844 UTC [48] LOG:  starting PostgreSQL 16.2 (Debian 16.2-1.pgdg120+2) on x86_64-pc-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
postgres    | 2024-02-16 04:03:25.915 UTC [48] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
postgres    | 2024-02-16 04:03:25.939 UTC [51] LOG:  database system was shut down at 2024-02-16 04:03:24 UTC
postgres    | 2024-02-16 04:03:25.948 UTC [48] LOG:  database system is ready to accept connections
postgres    |  done
postgres    | server started
postgres    | 
postgres    | /usr/local/bin/docker-entrypoint.sh: running /docker-entrypoint-initdb.d/init.sql
postgres    | CREATE TABLE
postgres    | CREATE TABLE
postgres    | INSERT 0 1
postgres    | INSERT 0 1
postgres    | INSERT 0 1
postgres    | INSERT 0 1
postgres    | INSERT 0 1
postgres    | 
postgres    | 
postgres    | waiting for server to shut down....2024-02-16 04:03:26.762 UTC [48] LOG:  received fast shutdown request
postgres    | 2024-02-16 04:03:26.812 UTC [48] LOG:  aborting any active transactions
postgres    | 2024-02-16 04:03:26.813 UTC [48] LOG:  background worker "logical replication launcher" (PID 54) exited with exit code 1
postgres    | 2024-02-16 04:03:26.814 UTC [49] LOG:  shutting down
postgres    | 2024-02-16 04:03:26.852 UTC [49] LOG:  checkpoint starting: shutdown immediate
postgres    | 2024-02-16 04:03:27.105 UTC [49] LOG:  checkpoint complete: wrote 64 buffers (0.4%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.022 s, sync=0.149 s, total=0.292 s; sync files=46, longest=0.078 s, average=0.004 s; distance=179 kB, estimate=179 kB; lsn=0/1517648, redo lsn=0/1517648
postgres    | 2024-02-16 04:03:27.109 UTC [48] LOG:  database system is shut down
postgres    |  done
postgres    | server stopped
postgres    | 
postgres    | PostgreSQL init process complete; ready for start up.
postgres    | 
postgres    | 2024-02-16 04:03:27.187 UTC [1] LOG:  starting PostgreSQL 16.2 (Debian 16.2-1.pgdg120+2) on x86_64-pc-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
postgres    | 2024-02-16 04:03:27.202 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
postgres    | 2024-02-16 04:03:27.202 UTC [1] LOG:  listening on IPv6 address "::", port 5432
postgres    | 2024-02-16 04:03:27.210 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
postgres    | 2024-02-16 04:03:27.218 UTC [64] LOG:  database system was shut down at 2024-02-16 04:03:27 UTC
postgres    | 2024-02-16 04:03:27.225 UTC [1] LOG:  database system is ready to accept connections
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-1 exited with code 0
deno-api-2 exited with code 0
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-1 exited with code 1
deno-api-2 exited with code 1
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-1 exited with code 1
deno-api-2 exited with code 1
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-1 exited with code 1
deno-api-2 exited with code 1
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-1 exited with code 1
deno-api-2 exited with code 1
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1 exited with code 1
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2 exited with code 1
caddy-1     | {"level":"error","ts":1708056216.8989372,"logger":"http.log.error","msg":"dial tcp 172.28.0.3:8080: i/o timeout","request":{"remote_ip":"172.28.0.1","remote_port":"57506","client_ip":"172.28.0.1","proto":"HTTP/1.1","method":"GET","host":"localhost:9999","uri":"/clientes/1/extrato","headers":{"User-Agent":["curl/7.81.0"],"Accept":["*/*"]}},"duration":11.284918836,"status":502,"err_id":"afr6jhdiz","err_trace":"reverseproxy.statusError (reverseproxy.go:1267)"}
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1 exited with code 1
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2 exited with code 1
caddy-1     | {"level":"error","ts":1708056225.0825458,"logger":"http.log.error","msg":"dial tcp: lookup deno-api-2 on 127.0.0.11:53: server misbehaving","request":{"remote_ip":"172.28.0.1","remote_port":"39610","client_ip":"172.28.0.1","proto":"HTTP/1.1","method":"GET","host":"localhost:9999","uri":"/clientes/1/extrato","headers":{"User-Agent":["curl/7.81.0"],"Accept":["*/*"]}},"duration":6.176882655,"status":502,"err_id":"93vskspks","err_trace":"reverseproxy.statusError (reverseproxy.go:1267)"}
caddy-1     | {"level":"error","ts":1708056233.2639852,"logger":"http.log.error","msg":"dial tcp: lookup deno-api-1 on 127.0.0.11:53: server misbehaving","request":{"remote_ip":"172.28.0.1","remote_port":"39612","client_ip":"172.28.0.1","proto":"HTTP/1.1","method":"GET","host":"localhost:9999","uri":"/clientes/1/extrato","headers":{"User-Agent":["curl/7.81.0"],"Accept":["*/*"]}},"duration":6.174555564,"status":502,"err_id":"96f3j6mqk","err_trace":"reverseproxy.statusError (reverseproxy.go:1267)"}
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1 exited with code 1
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2 exited with code 1
caddy-1     | {"level":"error","ts":1708056241.4373271,"logger":"http.log.error","msg":"dial tcp: lookup deno-api-2 on 127.0.0.11:53: server misbehaving","request":{"remote_ip":"172.28.0.1","remote_port":"48260","client_ip":"172.28.0.1","proto":"HTTP/1.1","method":"GET","host":"localhost:9999","uri":"/clientes/1/extrato","headers":{"User-Agent":["curl/7.81.0"],"Accept":["*/*"]}},"duration":6.166576029,"status":502,"err_id":"fn5axccjv","err_trace":"reverseproxy.statusError (reverseproxy.go:1267)"}
caddy-1     | {"level":"error","ts":1708056249.6161366,"logger":"http.log.error","msg":"dial tcp: lookup deno-api-1 on 127.0.0.11:53: server misbehaving","request":{"remote_ip":"172.28.0.1","remote_port":"37648","client_ip":"172.28.0.1","proto":"HTTP/1.1","method":"GET","host":"localhost:9999","uri":"/clientes/1/extrato","headers":{"User-Agent":["curl/7.81.0"],"Accept":["*/*"]}},"duration":6.172055897,"status":502,"err_id":"5iytt402h","err_trace":"reverseproxy.statusError (reverseproxy.go:1267)"}
caddy-1     | {"level":"error","ts":1708056257.7921152,"logger":"http.log.error","msg":"dial tcp: lookup deno-api-2 on 127.0.0.11:53: server misbehaving","request":{"remote_ip":"172.28.0.1","remote_port":"32874","client_ip":"172.28.0.1","proto":"HTTP/1.1","method":"GET","host":"localhost:9999","uri":"/clientes/1/extrato","headers":{"User-Agent":["curl/7.81.0"],"Accept":["*/*"]}},"duration":6.1694825,"status":502,"err_id":"ve5vwtwai","err_trace":"reverseproxy.statusError (reverseproxy.go:1267)"}
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1 exited with code 1
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2 exited with code 1
caddy-1     | {"level":"error","ts":1708056270.5729628,"logger":"http.log.error","msg":"dial tcp 172.28.0.3:8080: i/o timeout","request":{"remote_ip":"172.28.0.1","remote_port":"42528","client_ip":"172.28.0.1","proto":"HTTP/1.1","method":"GET","host":"localhost:9999","uri":"/clientes/1/extrato","headers":{"User-Agent":["curl/7.81.0"],"Accept":["*/*"]}},"duration":10.774069355,"status":502,"err_id":"vrf5n7b0b","err_trace":"reverseproxy.statusError (reverseproxy.go:1267)"}
caddy-1     | {"level":"error","ts":1708056278.746774,"logger":"http.log.error","msg":"dial tcp: lookup deno-api-1 on 127.0.0.11:53: server misbehaving","request":{"remote_ip":"172.28.0.1","remote_port":"42150","client_ip":"172.28.0.1","proto":"HTTP/1.1","method":"GET","host":"localhost:9999","uri":"/clientes/1/extrato","headers":{"User-Agent":["curl/7.81.0"],"Accept":["*/*"]}},"duration":6.166903876,"status":502,"err_id":"2yfm0yxha","err_trace":"reverseproxy.statusError (reverseproxy.go:1267)"}
caddy-1     | {"level":"error","ts":1708056286.917665,"logger":"http.log.error","msg":"dial tcp: lookup deno-api-2 on 127.0.0.11:53: server misbehaving","request":{"remote_ip":"172.28.0.1","remote_port":"47524","client_ip":"172.28.0.1","proto":"HTTP/1.1","method":"GET","host":"localhost:9999","uri":"/clientes/1/extrato","headers":{"Accept":["*/*"],"User-Agent":["curl/7.81.0"]}},"duration":6.16425339,"status":502,"err_id":"ngbh9ieuc","err_trace":"reverseproxy.statusError (reverseproxy.go:1267)"}
caddy-1     | {"level":"error","ts":1708056295.0871766,"logger":"http.log.error","msg":"dial tcp: lookup deno-api-1 on 127.0.0.11:53: server misbehaving","request":{"remote_ip":"172.28.0.1","remote_port":"41054","client_ip":"172.28.0.1","proto":"HTTP/1.1","method":"GET","host":"localhost:9999","uri":"/clientes/1/extrato","headers":{"User-Agent":["curl/7.81.0"],"Accept":["*/*"]}},"duration":6.162745554,"status":502,"err_id":"k4aczw73f","err_trace":"reverseproxy.statusError (reverseproxy.go:1267)"}
caddy-1     | {"level":"error","ts":1708056303.267494,"logger":"http.log.error","msg":"dial tcp: lookup deno-api-2 on 127.0.0.11:53: server misbehaving","request":{"remote_ip":"172.28.0.1","remote_port":"41066","client_ip":"172.28.0.1","proto":"HTTP/1.1","method":"GET","host":"localhost:9999","uri":"/clientes/1/extrato","headers":{"User-Agent":["curl/7.81.0"],"Accept":["*/*"]}},"duration":6.173461625,"status":502,"err_id":"zaa0q2aev","err_trace":"reverseproxy.statusError (reverseproxy.go:1267)"}
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1 exited with code 1
caddy-1     | {"level":"error","ts":1708056311.9471576,"logger":"http.log.error","msg":"dial tcp 172.28.0.3:8080: connect: connection refused","request":{"remote_ip":"172.28.0.1","remote_port":"57850","client_ip":"172.28.0.1","proto":"HTTP/1.1","method":"GET","host":"localhost:9999","uri":"/clientes/1/extrato","headers":{"User-Agent":["curl/7.81.0"],"Accept":["*/*"]}},"duration":6.672813322,"status":502,"err_id":"csn7mf5xf","err_trace":"reverseproxy.statusError (reverseproxy.go:1267)"}
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2 exited with code 1
caddy-1     | {"level":"error","ts":1708056320.1321547,"logger":"http.log.error","msg":"dial tcp: lookup deno-api-2 on 127.0.0.11:53: server misbehaving","request":{"remote_ip":"172.28.0.1","remote_port":"36406","client_ip":"172.28.0.1","proto":"HTTP/1.1","method":"GET","host":"localhost:9999","uri":"/clientes/1/extrato","headers":{"User-Agent":["curl/7.81.0"],"Accept":["*/*"]}},"duration":6.177928085,"status":502,"err_id":"7zamenx0e","err_trace":"reverseproxy.statusError (reverseproxy.go:1267)"}
caddy-1     | {"level":"error","ts":1708056328.3090084,"logger":"http.log.error","msg":"dial tcp: lookup deno-api-1 on 127.0.0.11:53: server misbehaving","request":{"remote_ip":"172.28.0.1","remote_port":"49964","client_ip":"172.28.0.1","proto":"HTTP/1.1","method":"GET","host":"localhost:9999","uri":"/clientes/1/extrato","headers":{"User-Agent":["curl/7.81.0"],"Accept":["*/*"]}},"duration":6.170002853,"status":502,"err_id":"2ud6mvee2","err_trace":"reverseproxy.statusError (reverseproxy.go:1267)"}
caddy-1     | {"level":"error","ts":1708056336.4715207,"logger":"http.log.error","msg":"dial tcp: lookup deno-api-2 on 127.0.0.11:53: server misbehaving","request":{"remote_ip":"172.28.0.1","remote_port":"42644","client_ip":"172.28.0.1","proto":"HTTP/1.1","method":"GET","host":"localhost:9999","uri":"/clientes/1/extrato","headers":{"Accept":["*/*"],"User-Agent":["curl/7.81.0"]}},"duration":6.155851649,"status":502,"err_id":"hjtuh7284","err_trace":"reverseproxy.statusError (reverseproxy.go:1267)"}
caddy-1     | {"level":"error","ts":1708056344.649285,"logger":"http.log.error","msg":"dial tcp: lookup deno-api-1 on 127.0.0.11:53: server misbehaving","request":{"remote_ip":"172.28.0.1","remote_port":"42660","client_ip":"172.28.0.1","proto":"HTTP/1.1","method":"GET","host":"localhost:9999","uri":"/clientes/1/extrato","headers":{"User-Agent":["curl/7.81.0"],"Accept":["*/*"]}},"duration":6.171074893,"status":502,"err_id":"0k5afjcwn","err_trace":"reverseproxy.statusError (reverseproxy.go:1267)"}
caddy-1     | {"level":"error","ts":1708056352.8382974,"logger":"http.log.error","msg":"dial tcp: lookup deno-api-2 on 127.0.0.11:53: server misbehaving","request":{"remote_ip":"172.28.0.1","remote_port":"49398","client_ip":"172.28.0.1","proto":"HTTP/1.1","method":"GET","host":"localhost:9999","uri":"/clientes/1/extrato","headers":{"User-Agent":["curl/7.81.0"],"Accept":["*/*"]}},"duration":6.182446942,"status":502,"err_id":"djuktkixt","err_trace":"reverseproxy.statusError (reverseproxy.go:1267)"}
caddy-1     | {"level":"error","ts":1708056361.010462,"logger":"http.log.error","msg":"dial tcp: lookup deno-api-1 on 127.0.0.11:53: server misbehaving","request":{"remote_ip":"172.28.0.1","remote_port":"35912","client_ip":"172.28.0.1","proto":"HTTP/1.1","method":"GET","host":"localhost:9999","uri":"/clientes/1/extrato","headers":{"User-Agent":["curl/7.81.0"],"Accept":["*/*"]}},"duration":6.165108917,"status":502,"err_id":"ukyuysdph","err_trace":"reverseproxy.statusError (reverseproxy.go:1267)"}
caddy-1     | {"level":"error","ts":1708056369.1913042,"logger":"http.log.error","msg":"dial tcp: lookup deno-api-2 on 127.0.0.11:53: server misbehaving","request":{"remote_ip":"172.28.0.1","remote_port":"59264","client_ip":"172.28.0.1","proto":"HTTP/1.1","method":"GET","host":"localhost:9999","uri":"/clientes/1/extrato","headers":{"User-Agent":["curl/7.81.0"],"Accept":["*/*"]}},"duration":6.174162981,"status":502,"err_id":"2yuj04jkb","err_trace":"reverseproxy.statusError (reverseproxy.go:1267)"}
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1  | exec ./main: exec format error
deno-api-1 exited with code 1
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2  | exec ./main: exec format error
deno-api-2 exited with code 1
caddy-1     | {"level":"error","ts":1708056377.337882,"logger":"http.log.error","msg":"dial tcp: lookup deno-api-1 on 127.0.0.11:53: server misbehaving","request":{"remote_ip":"172.28.0.1","remote_port":"44156","client_ip":"172.28.0.1","proto":"HTTP/1.1","method":"GET","host":"localhost:9999","uri":"/clientes/1/extrato","headers":{"Accept":["*/*"],"User-Agent":["curl/7.81.0"]}},"duration":6.140086727,"status":502,"err_id":"wur34i5if","err_trace":"reverseproxy.statusError (reverseproxy.go:1267)"}
caddy-1     | {"level":"info","ts":1708056379.4593525,"msg":"shutting down apps, then terminating","signal":"SIGTERM"}
caddy-1     | {"level":"warn","ts":1708056379.4593792,"msg":"exiting; byeee!! 👋","signal":"SIGTERM"}
caddy-1     | {"level":"info","ts":1708056379.4594135,"logger":"http","msg":"servers shutting down with eternal grace period"}
caddy-1     | {"level":"info","ts":1708056379.4595144,"msg":"shutdown complete","signal":"SIGTERM","exit_code":0}
caddy-1 exited with code 0
caddy-1 exited with code 0
deno-api-2 exited with code 0
deno-api-1 exited with code 0
postgres    | 2024-02-16 04:06:20.031 UTC [1] LOG:  received fast shutdown request
postgres    | 2024-02-16 04:06:20.036 UTC [1] LOG:  aborting any active transactions
postgres    | 2024-02-16 04:06:20.038 UTC [1] LOG:  background worker "logical replication launcher" (PID 67) exited with exit code 1
postgres    | 2024-02-16 04:06:20.038 UTC [62] LOG:  shutting down
postgres    | 2024-02-16 04:06:20.041 UTC [62] LOG:  checkpoint starting: shutdown immediate
postgres    | 2024-02-16 04:06:20.105 UTC [62] LOG:  checkpoint complete: wrote 17 buffers (0.1%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.014 s, sync=0.031 s, total=0.068 s; sync files=12, longest=0.006 s, average=0.003 s; distance=26 kB, estimate=26 kB; lsn=0/151E180, redo lsn=0/151E180
postgres    | 2024-02-16 04:06:20.109 UTC [1] LOG:  database system is shut down
postgres exited with code 0
